docs/api/resource.rst

```diff
@@ -351,10 +353,10 @@ GET /{collection}/<id>
 
 Returns a specific record by its id.
 
-For convenience and consistency, a header ``Last-Modified`` will also repeat the
+For convenience and consistency, a header ``ETag`` will also repeat the
 value of the ``last_modified`` field.
```

Comment:
So far the philosophy of this toolkit has been to provide an implementation of a particular protocol. But you are right, ETags should remain «*opaque*» and shouldn't have too much semantics. 
I'll remove that sentence about «*repeating the last modified value*».

Thanks !